### ISSUE TITLE
Question regarding transfer learning approach

### ISSUE THREAD

**Issue by kevinkovalchik** [bzhanglab/DeepRescore#1]
Hello!

Thanks for developing this tool. It is nice to see this rescoring approach finally packaged in an easily distributed and reproducible manner.

I have a question regarding the fragment ion intensity prediction model. Are you using the pre-trained model provided with pDeep2 as it is? Or is the model fine-tuned using the high-confidence PSMs (i.e. like AutoRT is) before predicting all the spectra?

Best wishes,
Kevin

**Comment by wenbostar** [comment]
Hi Kevin,

Since MS/MS spectra generated from the same type of MS spectrometers or similar MS spectrometers with similar parameters are very similar, we just use the pre-trained models provided by pDeep2 for MS/MS spectrum prediction. Fine-tuning the pDeep2 models using the high-confidence PSMs may further improve the performance of rescoring. But we haven't tried that yet.

Bo

**Comment by kevinkovalchik** [comment]
Hi Bo,

Sorry for my super-slow response. I agree that instrument-specific differences wouldn't have a big impact on accuracy. I was thinking more about sample differences. My understanding is the pDeep models are trained on tryptic digests, and if someone is working with a different enzyme or with unspecific digests then the terminal peptides could be quite different from anything the model was trained on. From what I have seen models that predict fragment ions tend to generalize well enough to still do a good job, but it would be interesting to see if the transfer-training would help out.

Kevin
